billing/v1/billing.proto

```diff
@@ -1357,4 +1365,21 @@ message UpdateInvoiceServiceDiscountsRequest {
 message DeleteInvoiceServiceDiscountsRequest {
   // Required. [blueapi.api.ripple.v1.InvoiceServiceDiscounts.id]
   string id = 1;
-}
\ No newline at end of file
+}
+
+// Response for tag request
+message Customer {
+  string customerId = 1;
+  string accountId = 2;
+  string customerName = 3;
+  repeated Tags tags = 4;
+}
```

Comment:
@capoyon 
With BlueAPI's specs, it would be better to change from `customerId` to `id`, `customerName` to `name`

Reply:
It is applied, thanks